custom-tractusx-edc/custom-edc-controlplane-postgresql-hashicorp-vault/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+/********************************************************************************
+ *  Copyright (c) 2024 SAP SE
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       SAP SE - initial API and implementation
+ *
+ ********************************************************************************/
+
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+
+plugins {
+    `java-library`
+    id("application")
+    id("com.github.johnrengelman.shadow") version "8.1.1"
+}
+
+dependencies {
+    runtimeOnly(libs.edc.controlplane.postgresql.hashicorp.vault) {
+        exclude(module = "tx-iatp-sts-dim")
+    }
+    runtimeOnly(libs.edc.identity.trust.sts.remote.client)
+    runtimeOnly(libs.edc.auth.oauth2.client)
+}
+
+tasks.withType<ShadowJar> {
+    exclude("**/pom.properties", "**/pom.xm")
+    mergeServiceFiles()
+    archiveFileName.set("${project.name}.jar")
+}
+
+application {
+    mainClass.set("org.eclipse.edc.boot.system.runtime.BaseRuntime")
+}
+
+// do not publish
+edcBuild {
+    publish.set(false)
+}
```

Comment:
also this is not needed because we're not publishing any jar artifact here anyway